make-release.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+#
+# Copyright (c) 2020 Red Hat, Inc.
+# This program and the accompanying materials are made
+# available under the terms of the Eclipse Public License 2.0
+# which is available at https://www.eclipse.org/legal/epl-2.0/
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#   Red Hat, Inc. - initial API and implementation
+
+# Based on https://github.com/che-incubator/chectl/blob/master/make-release.sh
+
+set -e
+set -u
+
+usage ()
+{   echo "Usage: ./make-release.sh <schema-version> <k8s-api-version>"
+    exit
+}
+
+if [[ $# -lt 2 ]]; then usage; fi
+
+if ! command -v hub > /dev/null; then
+  echo "[ERROR] The hub CLI needs to be installed. See https://github.com/github/hub/releases"
+  exit
+fi
+if [[ -z "${GITHUB_TOKEN}" ]]; then
+  echo "[ERROR] The GITHUB_TOKEN environment variable must be set."
+  exit
+fi
+
+VERSION=$1
+API_VERSION=$2
+
+init() {
+  BRANCH=$(echo $VERSION | sed 's/.$/x/')
```

Comment:
Not a huge issue now, but this `sed` replacement will fail if we ever get to e.g. `2.0.10`. Maybe something like `sed 's|[0-9]\+$|x|'` is safer?

Reply:
Yeah, good idea.

Reply:
I ended up not using sed for this, and took a separate approach that's a little cleaner I think.